src/app/main/chat/chat.component.ts

```diff
@@ -7,9 +11,20 @@
 })
 export class ChatComponent implements OnInit {
 
-  constructor() { }
+  constructor(
+    private _api : ApiService,
+    private _auth: AuthService,
+    private router: Router,
+    public fb: FormBuilder
+  ) { }
 
   ngOnInit(): void {
+    this._api.getTypeRequest('chat').subscribe((res: any) => {
+      console.log(res)
+    }, err => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/ticodacytest/chatApp1/pullRequest?prid=7784558)
```suggestion
    }, (err) => {
```